recipes/ringdecomposerlib/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "RingDecomposerLib" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rareylab/RingDecomposerLib/archive/refs/tags/v{{ version }}_rdkit.tar.gz
```

Comment:
What is the meaning of `rdkit` here? It's part of the version number it seems.

Reply:
I guess this [RDKit](https://www.rdkit.org/) is meant, not sure why the label was included in tag. The CMake build file sets for all available releases (1.1, 1.1.1, 1.1.2, and 1.1.3(\_rdkit)) the version 1.0

Reply:
There is no 1.1.3 release without the label and I think including the label in the version will confuse the conda/mamba.